bc/web/views.py

```diff
@@ -18,3 +20,29 @@ def count_dockets(request: HttpRequest) -> HttpResponse:
     return TemplateResponse(
         request, "homepage.html", {"subscription_count": subscription_count}
     )
+
+
+def little_cases(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = BotSuggestionForm(request.POST)
+        if form.is_valid():
+            form.save()
+            request.session["suggestion_submitted"] = True
```

Comment:
I think the better way to do this is to just redirect somebody to another page that says thank you but nothing else. 

This approach will always have the value be True (even after refresh or browsing somewhere else).

I'm also not sure if we support cookies on the website properly at the moment. It could certainly be that cookies are part of the CDN key, meaning that one person's submission (and cookies) could get caught in the CDN and shown to others. I guess we'll want to check on that, but in the meantime, we can just redirect to a thank you page.

Reply:
Thanks for your suggestion. I will add a new page to redirect the users after they submit a form.